cmd/utils/flags.go

```diff
@@ -1093,6 +1093,13 @@ func setBootstrapNodes(ctx *cli.Context, cfg *p2p.Config) {
 			urls = params.SepoliaBootnodes
 		case ctx.Bool(GoerliFlag.Name):
 			urls = params.GoerliBootnodes
+		case ctx.IsSet(OPNetworkFlag.Name):
+			network := ctx.String(OPNetworkFlag.Name)
+			if strings.Contains(strings.ToLower(network), "mainnet") {
+				urls = params.OPMainnetBootnodes
+			} else {
+				urls = params.OPSepoliaBootnodes
+			}
```

Comment:
Do you on purpose default to sepolia bootnodes, instead of explicitly checking for "sepolia" in the network name? Goerli isn't dead yet... same below.

Reply:
yes, we fallback to to sepolia bootnodes for everything. With V5, it can work cross network